nimble/host/src/ble_sm.c

```diff
@@ -2910,4 +2910,109 @@ ble_sm_create_chan(uint16_t conn_handle)
     return chan;
 }
 
+#if MYNEWT_VAL(BLE_SM_SC)
```

Comment:
I'd add `BLE_SM_CSIS_SIRK` to enable everything related to handling CSIS SIRK and move this to `ble_sm_csis.c`

also mark syscfg as experimental since we may want to move it to some place more appropriate and rename in future with addition of le audio code.